Library/Homebrew/cmd/update.rb

```diff
@@ -442,28 +442,23 @@ def select_formula(key)
   def dump_formula_report(key, title)
     formula = select_formula(key)
     unless formula.empty?
-      # Determine list item indices of installed formulae.
-      formula_installed_index = formula.each_index.select do |index|
-        name, newname = formula[index]
-        installed?(name) || (newname && installed?(newname))
+      # Make installed formulae more visible/pretty.
+      formula.map! do |name, newname|
+        name = pretty_installed(name) if installed?(name)
+        newname = pretty_installed(newname) if (newname && installed?(newname))
+        [name, newname]
       end
 
       # Format list items of renamed formulae.
       if key == :R
         formula.map! { |oldname, newname| "#{oldname} -> #{newname}" }
```

Comment:
That's not defently true. Since migration happens later and won't guarantee to be success.